webrtc.html

```diff
@@ -4625,16 +4629,21 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
       source, as is the case for a <code>MediaStream</code> received over an
       <code><a>RTCPeerConnection</a></code> ) is always strong.</p>
 
-      <p>A <code><a>MediaStreamTrack</a></code>, received from the
-      <code><a href="#event-track">ontrack</a></code> event on an
-      <code><a>RTCPeerConnection</a></code>, MUST have its
-      <code>muted</code> attribute [[!GETUSERMEDIA]] set to
-      <code>true</code> until media data arrives from the RTP source.</p>
-
-      <p>In addition, a <code>MediaStreamTrack</code> has its
-      <code>muted</code> attribute set to <code>true</code> on the
-      remote peer if the local user agent makes inactive the corresponding
-      <code><a>MediaStreamTrack</a></code> that is being sent.</p>
+      <p>When an <code><a>RTCPeerConnection</a></code> receives data on an RTP
+      source for the first time, it MUST <a href="#update-track-muted">update
+      the muted state</a> of the corresponding <code><a>MediaStreamTrack</a>
+      </code> with the value <code>false</code>.</p>
+
+      <p>When an <code><a>RTCPeerConnection</a></code>'s RTP source is
```

Comment:
This is from getusermedia - the muted state means that no data is coming, but the track is not promising that it's permanent - so it's not ended.
It could be for a very long time. Does it matter to any of the rest of the spec how long it is?